source/main/modules/window-manager/index.ts

```diff
@@ -844,7 +917,7 @@ export default class WindowManager {
     * @param  {MDFileDescriptor|DirDescriptor} descriptor The corresponding descriptor
     * @return {boolean}                                   True if user wishes to remove it.
     */
-  async confirmRemove (descriptor: MDFileDescriptor|CodeFileDescriptor|DirDescriptor): Promise<boolean> {
+  async confirmRemove (descriptor: MDFileDescriptor | CodeFileDescriptor | DirDescriptor): Promise<boolean> {
```

Comment:
Don't make whitespace changes to code unrelated to the pull request. I don't think the Zettlr developers will approve the whitespace changes.